docusaurus/docs/migration-guides/update-from-grafana-versions/v7.x-v8.x.md

```diff
@@ -2,7 +2,14 @@
 id: migrate-7.x.x-to-8.x.x
 title: 7.x.x to 8.x.x
 sidebar_position: 7
-description: "Follow these instructions to migrate Grafana v7.x.x plugins to the updated plugin system available in Grafana v8.x.x."
+description: How to migrate Grafana v7.x.x plugins to the updated plugin system available in Grafana v8.x.x.
```

Comment:
Maybe just 7.x to 8.x? Same for title. Thinking wrt consistency with v6 docs?

Reply:
Fixed.